frontend_lib/src/component/NewMemberForm/NewMemberForm.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import { translate } from 'react-i18next'
 import Avatar from '../Avatar/Avatar.jsx'
+import WarningIcon from '../Icon/WarningIcon'
```

Comment:
without '.jsx' the path is getting resolved. It works because of webpack does resolves the .js and .jsx files.
But we usually don't use it in Tracim, so please add the extension to the file